api/api/clubs/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class Club (models.Model):
+    CULTURAL_CLUB = 'Cultural'
+    SCIENCE_AND_TECH_CLUB = 'Science & Technology'
+    SPORTS_CLUB = 'Sports'
+
+    CLUB_TYPE_CHOICES = [
+        ('Cultural', CULTURAL_CLUB),
+        ('S&T', SCIENCE_AND_TECH_CLUB),
+        ('Sports', SPORTS_CLUB)
```

Comment:
Is there an upside to keeping these as variables? `CULTURAL_CLUB`, `SCIENCE_AND...` etc.

Reply:
I suppose use of variables increases code readability and provides single point of change if necessary.

Reply:
For readability: CULTURAL_CLUB as a string is also pretty readable

But yes, single point of change in variables is useful.

Reply:
So should I change or keep the variables .